### PR TITLE
Colors and ux updates

### PR DESCRIPTION
**Pull request by davidendico-owkin**
Updated Substra documentation according the new branding
- Changed logotype
- Updated fonts
- Updated colors
- Enhanced overall look and feel
- Improved responsive

### CODE REVIEW

File: docs/source/static/owkin.css [hunk 1]
```diff
@@ -1,97 +1,181 @@
...
+
+/* Font face */
+
+@font-face {
+    font-family: "Gattica";
+    src: url(Gattica-Light100.otf) format("opentype");
+    font-weight: 400;
+}
+
+@font-face {
```
Comment (Milouu) [inline docs/source/static/owkin.css:12]: You've put Regular 2 times :)

File: docs/source/static/owkin.css [hunk 2]
```diff
@@ -1,97 +1,181 @@
...
-    --color-blue: #37b3ff;
-    --color-gold: #ffc200;
-    --color-lightest-gray: #f5f5f5;
-    --color-light-gray: #d9d9d9;
-    --color-light-gold: #faf3eb;
+/* VARIABLES */
+
+/* Font face */
+
+@font-face {
```
Comment (Milouu) [inline docs/source/static/owkin.css:6]: To be iso with what we have  as font-weights in frontend, should be light: 100, regular: 400, medium: 500, bold: 700

**Review by Milouu — CHANGES_REQUESTED** [review]

**Review by ThibaultFy — COMMENTED** [review]
Thanks for the PR :)
The text f the doc is now in center of the page instead of the left. Is it on purpose ?

File: docs/source/static/owkin.css (hunk 2 above)
Comment (davidendico-owkin) [reply docs/source/static/owkin.css:6]: You are right! I'm on it!

File: docs/source/static/owkin.css (hunk 1 above)
Comment (davidendico-owkin) [reply docs/source/static/owkin.css]: Done!

File: docs/source/static/owkin.css (hunk 2 above)
Comment (davidendico-owkin) [reply docs/source/static/owkin.css:6]: Seems to be good now :)

**Comment by davidendico-owkin** [comment]
> 

Ahah, tastes and colors 😌 In this case adding more space between the navigation and the content tends to make them both more identifiable and easy to scan (and I also had multiple feedbacks about the weird unused white space at the right of the screen)! So if you're ok with it, let's try this way!

**Comment by Milouu** [comment]
Something must have gone wrong during rebase because i see the `chore: release 0.22` as one of your commit and the code is mixed with yours

**Comment by ThibaultFy** [comment]
> Something must have gone wrong during rebase because i see the `chore: release 0.22` as one of your commit and the code is mixed with yours

I fixed it, @davidendico-owkin git pull your branch before doing other changes if needed :)

**Review by Milouu — APPROVED** [review]

**Review by RomainGoussault — APPROVED** [review]
Thanks @davidendico-owkin this is beautiful 🤩

**Review by ThibaultFy — APPROVED** [review]
LGTM, thanks a lot for the PR !! 😄

**Pull request merged**